README.md

```diff
@@ -3,46 +3,3 @@
 This repo contains a number of [Hail](https://hail.is/) utility functions and scripts for the [gnomAD project](http://gnomad.broadinstitute.org) and the [MacArthur lab](http://macarthurlab.org). As we continue to expand the size of our datasets, we are constantly seeking to find ways to reduce the complexity of our workflows and to make these functions more generic. As a result, the interface for many of these functions will change over time as we generalize their implementation for more flexible use within our scripts. We are also continously adapting our code to regular changes in the Hail interface. These repos thus represent only a snapshot of the gnomAD code base and are shared without guarantees or warranties.
 
 We therefore encourage users to browse through the [API reference](https://macarthur-lab.github.io/gnomad_hail/api_reference/) to identify modules and functions that will be useful in their own pipelines, and to edit and reconfigure relevant code to suit their particular analysis and QC needs.
```

Comment:
Will this github.io url `(https://macarthur-lab.github.io/gnomad_hail/api_reference/) ` need to be changed once the repo is renamed ?

Reply:
Yes. Leaving that change for @gtiao.